### DISCUSSION TITLE
SelectOption `valueText` prop

### DISCUSSION THREAD

**Discussion by jiblett1000** [chakra-ui/ark#847] [Q&A]
Just hoping for some clarification on the distinction between the `SelectOption` props (i.e. label, value, valueText). I'm guessing it's as follows:

**label**: The text displayed on the trigger / button element when an option is selected and for each said item in the list.
**value**: The actual value of the option which is normally not displayed.
**valueText**: An optional text which is displayed in the options list for each item (instead of `label`), but `label` can still be used for display in the trigger element. This allows the two values to be different.

Is that understanding correct?

**Comment by wencyen** [comment]
which version of `Select` are you using and for which framework?
